### PR TITLE
Sound for 8470p 

### PR DESCRIPTION
**Pull request by dreadkopp**
should work for 8460p as well,

couldn't get sound with the original scripts and High Sierra.

DP out is working by changing Platform id to 0166004 and applying framebuffer kext patch for lvds. sound is gone then though.... will investigate further

### CODE REVIEW

**Comment by RehabMan** [comment]
Audio on the 8460p works if you follow the guide.
Use the forums if you need help.

**Comment by YeshuaChrist** [comment]
I found a post installation software for 8460p for high sierra. Will that software work with 8470p with intel hd 4000 computers? How similar are 8460p and 8470p? I am not that smart and these text guides to me are hard to follow. If you made a video for dumb butts like me i'd appreciate that more because I guess I'm slightly dyslexic in some ways when it comes to reading. ;(

**Comment by RehabMan** [comment]
8470 is 7-series/HD4000.
8460 is 6-series/HD3000.
Both are covered by the same guide:
https://www.tonymacx86.com/threads/guide-hp-probook-elitebook-zbook-using-clover-uefi-hotpatch.232948/
